### ISSUE TITLE
thing1.swap(thing2)  'Thing' object does not support indexing

### ISSUE THREAD

**Issue by tamhv** [bfirsh/django-ordered-model#135]
I'm using this form to swap two object Thing, but i got error "'Thing' object does not support indexing"
```
class Thing(OrderedModel):
        ...

class ChangeOrderForm(forms.Form):
    task_source = forms.ModelChoiceField(queryset=Thing.objects.can_order_filter())
    task_target = forms.ModelChoiceField(queryset=Thing.objects.can_order_filter())

    def swap(self):
        task_source = self.cleaned_data.get('task_source')
        task_target = self.cleaned_data.get('task_target')
        if not (task_source == task_target):
           task_source.swap(task_target)
```

**Comment by tamhv** [comment]
Just change to this, it works!
```
    def swap(self):
        task_source = self.cleaned_data.get('task_source')
        task_target = self.cleaned_data.get('task_target')
        a = task_source.order
        b = task_target.order

        task_source.to(b)
        task_target.to(a)
        return True
```

**Comment by shuckc** [comment]
Hi. We cleaned up the behaviour of `swap()` in #131 a month ago, but as it's a breaking change so needs to wait for the v2.0 release.  If you read through that ticked you will see why swap doens't work as it should in the version you are using.

**Comment by shuckc** [comment]
You could get your original code working by installing pre-release from git, e.g. with
```
$ pip uninstall django-ordered-model
$ pip install git+git://github.com/bfirsh/django-ordered-model.git
```

**Comment by rpkilby** [comment]
Is this now resolved since 2.0 has been released?

**Comment by shuckc** [comment]
Yes it's resolved in all recent releases.
